package.json

```diff
@@ -24,7 +24,7 @@
     "libs/**/*"
   ],
   "dependencies": {
-    "pxt-core": "^7.5.19"
+    "pxt-core": "7.5.23"
```

Comment:
@jwunderl @riknoll @abchatra I changed this to be an exact version match. This is how it is in pxt-arcade. Was there a reason for caret matching?

Reply:
Not that I'm aware of. This isn't a super important dependency anyhow! it's really just for typings

Reply:
Change looks good to me. As Richard said not very important dependency as it doesn't impact the target build.